backend/utilities/csvObjectParse.js

```diff
@@ -0,0 +1,16 @@
+csvParseObject = object => {
+  let newObject = {};
+  Object.keys(object).forEach(key => {
+    let newKey;
+    Object.keys(object[key]).forEach(subKey => {
+      if (subKey === "Id") {
+        newKey = object[key][subKey];
+      } else {
+        newObject[newKey][subKey] = object[key][subKey];
+      }
+    });
+  });
+  return newObject;
+};
+
+module.exports = csvParseObject;
```

Comment:
Really liking these utilities! This is a good example of abstraction, like my monster comment talks about.